airflow/jobs/scheduler_job.py

```diff
@@ -556,12 +556,15 @@ def execute_callbacks(
         :param session: DB session.
         """
         for request in callback_requests:
-            if isinstance(request, TaskCallbackRequest):
-                self._execute_task_callbacks(dagbag, request)
-            elif isinstance(request, SlaCallbackRequest):
-                self.manage_slas(dagbag.dags.get(request.dag_id))
-            elif isinstance(request, DagCallbackRequest):
-                self._execute_dag_callbacks(dagbag, request, session)
+            try:
+                if isinstance(request, TaskCallbackRequest):
+                    self._execute_task_callbacks(dagbag, request)
+                elif isinstance(request, SlaCallbackRequest):
+                    self.manage_slas(dagbag.dags.get(request.dag_id))
```

Comment:
Just a nit, but as we are refactoring code here should we make `manage_slas` private too?

Reply:
I think `manage_slas` is fine being public since you can run perform some actions and tests outside of the Scheduler too. The only reason it is not a static method is because we use self.log